README.md

```diff
@@ -39,12 +39,16 @@ Commit your code regularly and use descriptive messages. This helps both you (in
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] What is the purpose of using _sessions_?
+    To make sure that the information stays the same for each new request instead of reentering each time
 
 - [ ] What does bcrypt do to help us store passwords in a secure manner.
+    It uses a plain-text algorithm to basically hash the passwords into strings so that real passwords are not being stored 
```

Comment:
good answer...what does it use to do that? Does it just do one round of hashing or multiple? What is that called?